src/vmware/azext_vmware/_params.py

```diff
@@ -134,3 +134,11 @@ def load_arguments(self, _):
         c.argument('out', help='Standard output stream from the powershell execution.')
         c.argument('named_outputs', action=ScriptExecutionNamedOutputAction, nargs='*', help='User-defined dictionary.')
         c.argument('script_cmdlet_id', help='A reference to the script cmdlet resource if user is running a AVS script.')
+
+    with self.argument_context('vmware workload-network dhcp') as c:
+        c.argument('dhcp_id', help='NSX DHCP identifier. Generally the same as the DHCP display name.')
+        c.argument('display_name', help='Display name of the DHCP entity.')
+        c.argument('revision', help='NSX revision number.')
+        c.argument('server_address', help='DHCP Server Address.')
+        c.argument('lease_time', help='DHCP Server Lease Time.')
+        c.argument('server_addresses', nargs='*', help='DHCP Relay Addresses. Max 3.')

```

Comment:
How do you pass in multiple server addresses with it like this? What about using an `options_list`? It should be `nargs='+'` since the minimum is 1.